v3.0/how-to/deploy/tidb-binlog.md

```diff
@@ -56,7 +56,7 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
         172.16.10.74
         ```
 
-        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml` 文件中 `gc` 变量值，并取消注释。
+        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump-cluster.yml` (旧版本为pump.yml)文件中 `gc` 变量值，并取消注释。
```

Comment:
```suggestion
        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump-cluster.yml`（TiDB 3.0.1 及之前版本中为 `pump.yml`）文件中 `gc` 变量值，并取消注释。
```

Reply:
Please also help update this file in `dev`: `dev/how-to/deploy/tidb-binlog.md`.

Reply:
In v3.0.0 v3.0.1 v3.0.2 version is `pump-cluster.yml`, after v3.0.3 version changed to `pump.yml`.

Reply:
@liubo0127 Got it! 
@yilongrong I updated my suggestion based on liubo's comment. Please address the comment and also apply changes to `dev/how-to/deploy/tidb-binlog.md`.